core/lib/zksync_core/src/l1_gas_price/gas_adjuster/mod.rs

```diff
@@ -133,6 +166,55 @@ impl GasAdjuster {
         // For now, pubdata is only sent via calldata, so its price is pegged to the L1 gas price.
         self.estimate_effective_gas_price() * L1_GAS_PER_PUBDATA_BYTE as u64
     }
+
+    async fn get_blob_base_fee_history(
```

Comment:
what this will return when we roll it out to mainnet before Dencun upgrade ? Is it going to return empty vector ? what would be our estimate then ?

(please also put the answer in the function comment)

Reply:
Yes, empty vector. We must estimate pubdata price based on l1_gas_price until Dencun